demo/site/src/app/[visibility]/[domain]/graphql/route.ts

```diff
@@ -77,10 +78,16 @@ async function handler(req: Request) {
         }
     }
 
+    const preview = await previewParams({ skipDraftModeCheck: true });
+
     // Forward to actual GraphQL server
     const upstreamRes = await fetch(GRAPHQL_TARGET, {
         method: "POST",
-        headers: { "Content-Type": "application/json" },
+        headers: {
+            "Content-Type": "application/json",
+            Authorization: `Basic ${Buffer.from(`system-user:${process.env.API_BASIC_AUTH_SYSTEM_USER_PASSWORD}`).toString("base64")}`,
```

Comment:
using system user here is correct, as only persisted queries that are found in code are supported (how did this work before?)

Reply:
Probably due to the static users in the API.